tools/c7n_gcp/c7n_gcp/actions/core.py

```diff
@@ -70,7 +70,28 @@ def process_resource_set(self, client, model, resources):
         for resource in resources:
             op_name = self.get_operation_name(model, resource)
             params = self.get_resource_params(model, resource)
-            result = self.invoke_api(client, op_name, params)
+            try:
+                result = self.invoke_api(client, op_name, params)
+            except HttpError as e:
+                if 'fingerprint' in e.reason:
```

Comment:
I think we need to move this logic to a subclass for gke, perhaps having this base class delegate to a handle_exception hook, which defaults to raising.

Reply:
I don't disagree, but that's a bit beyond my knowledge of python (right now). I DID clean it up so that the individual models now have the refetchFingerprints method that allows us to define it per-model now and no have the logic for "what kind of resource is this" - if the model supports the "refetchFingerprint" function, it gets run, otherwise it raises the parent exception

Reply:
Looking at the future: to add label support to other resources, the changes needed would be: enable labels and add refetchFingerprint function to the resource, then test.